### PR TITLE
Modifies README.md

### PR DESCRIPTION
**Pull request by saramissak**

### CODE REVIEW

**Comment by saramissak** [comment]
https://github.com/saramissak/my-portfolio/pull/1/commits/a5479254a9a18fd8f5796d776b802a446a5b25ad

In this file, I just updated the README.md.

File: portfolio/src/main/webapp/contact.html [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    </div>
+
+    <!-- Third (White) container -->
+    <div class='container'>
+      <div class='section'>
+        <div class='row'>
+          <div class='col s12 center'>
+        	<h3><i class='mdi-content-send brown-text'></i></h3>
+	        <h4>Send me a message!</h4>
+            <form class="col s12" onsubmit="return validate()" action="mailto:sarammissak@gmail.com" method="post" enctype="text/plain">
```
Comment (saramissak) [inline portfolio/src/main/webapp/contact.html]: Is there a better way to do this? This way it opens the client's email and they have to manually send the email. I searched it up and it says you can only do it with PHP but I was wondering if you had any other ideas to this?

File: portfolio/src/main/webapp/LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2014-2018 Materialize
```
Comment (saramissak) [inline portfolio/src/main/webapp/LICENSE]: This is the copyright license of the framework I used

**Review by saramissak — COMMENTED** [review]
Hi could you please review my changes?

File: portfolio/src/main/webapp/LICENSE (hunk 2 above)
Comment (shenqiuz) [reply portfolio/src/main/webapp/LICENSE]: It is good that you include a copyright license file in your project. I probably will create a folder "license" and put all license files there. You can replace the file name with the framework's name. So it is clear that this license file is related to that framework. Sometimes when you release your product, you might need to merge all license files into a big license file. We have some internal tools to do that. But we can just use the license folder for this project.

Comment (saramissak) [reply portfolio/src/main/webapp/LICENSE]: Thank you for the recommendation. I made a folder and moved it there. There is also a license at the top of the java file and script.js file that came with the starter code. Should I move that into a file and place that under the license folder as well? I think I should, but I don't know if there is a difference when it is at the top of the file verses in a separate file.

Comment (shenqiuz) [reply portfolio/src/main/webapp/LICENSE]: For the licenses at the top of the java file and script.js file, we should just leave them there. So people knows that this file is protected by a license which usually doesn't contain all information (e.g. distribute/modify rights). When the people needs more information, they can check the detailed information in the license folder.

Comment (saramissak) [reply portfolio/src/main/webapp/LICENSE]: Okay. I understand the difference. Thank you!

File: portfolio/src/main/webapp/contact.html [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+                <div class="input-field col s6">
+                  <input id="lastName" type="text" class="validate" name="Last Name" required>
+                  <label for="lastName">Last Name</label>
+                </div>
+              </div>
+  
+              <div class="row">
+                <div class="input-field col s12">
+                  <i class="material-icons prefix">phone</i>
+                  <input id="telephone" type="tel" class="validate" maxlength="10" name="Phone Number" required>
```
Comment (shabbirahussain) [inline portfolio/src/main/webapp/contact.html:89]: Fixed length of 10 does not allow for country code. Also it prohibits users from copy-pasting phone numbers which might have separators in it. Our goal is to have users enter something valid without hampering their experience of the page.

An option here could be use a javascript based validator which validates the number on lost focus.

File: portfolio/src/main/webapp/js/script.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  factContainer.innerText = fact;
+}
+
+function validate() {
+    var phone = $("#telephone").val();
+    if (phone.length != 10 || isNaN(phone)) {
+        alert("Phone  number is not valid or not valid format.");
+        return false;
+    }
+}
```
Comment (shabbirahussain) [inline portfolio/src/main/webapp/js/script.js]: Please fix the linter error complaining about no new line at the end of the file. Generally these are there to promote a standardized code.

**Review by shabbirahussain — APPROVED** [review]

**Pull request merged**